registrar/apps/enrollments/models.py

```diff
@@ -0,0 +1,126 @@
+"""
+Models relating to learner program and course enrollments.
+"""
+from django.db import models
+from model_utils.models import TimeStampedModel
+from simple_history.models import HistoricalRecords
+
+
+class Organization(TimeStampedModel):
+    """
+    Model that represents a course-discovery Organization entity.
+    """
+    catalog_uuid = models.UUIDField(db_index=True, null=True)
+    name = models.CharField(max_length=255)
+    key = models.CharField(max_length=255)
+
+
+class Program(TimeStampedModel):
+    """
+    Table that referrences a course-discovery Program entity.
+    """
+    class Meta(object):
+        app_label = 'enrollments'
+
+    catalog_uuid = models.UUIDField(db_index=True, null=True)
```

Comment:
Should we rename this to discovery_uuid to not confuse it with the actual course catalog concept?

Reply:
sure.